doc/whats_new/v1.3.rst

```diff
@@ -350,6 +353,9 @@ Changelog
   :pr:`25733` by :user:`Brigitta Sipőcz <bsipocz>` and
   :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :func:`utils.validation.check_array` now suports extension arrays with object
```

Comment:
```suggestion
- |Fix| :func:`utils.validation.check_array` now supports extension arrays with object
```

Reply:
What are extension arrays? Maybe a link to pandas docs or just adding „pandas“ in the sentence.